packages/core-bridge/src/conversions.rs

```diff
@@ -176,38 +177,25 @@ impl ObjectHandleConversionsExt for Handle<'_, JsObject> {
 
     fn as_telemetry_options(&self, cx: &mut FunctionContext) -> NeonResult<TelemetryOptions> {
         let mut telemetry_opts = TelemetryOptionsBuilder::default();
-
-        if let Some(tf) = js_optional_value_getter!(cx, self, "tracingFilter", JsString) {
-            telemetry_opts.tracing_filter(tf);
-        }
```

Comment:
Didn't we deprecate `tracingFilter`? Looks like it is ignored here.

Reply:
Backward compatibility is dealt in TypeScript land (runtime.ts). Same for deprecated level option.